.travis.yml

```diff
@@ -2,6 +2,9 @@ language: erlang
 
 otp_release:
   - 18.0
+  - 18.1
+  - 18.2.1
```

Comment:
probably yes.
I originally suggested 18.2.1 because it looks like a bugfix release to solve an issue in windows.
http://www.erlang.org/downloads/18.2.1

I don't think by running 18.2.1 there's any need to run 1.8.2,